hazelcast/src/main/java/com/hazelcast/jet/impl/operation/TerminateJobOperation.java

```diff
@@ -74,4 +85,16 @@ protected void readInternal(ObjectDataInput in) throws IOException {
         terminationMode = TerminationMode.values()[in.readByte()];
         isLightJob = in.readBoolean();
     }
+
+    private boolean checkJobIsSuspendable(JobCoordinationService jcs) {
```

Comment:
name of this method is inconsistent with what it does

Reply:
still wrong. according to it's name the method should check only job config, but does more (checks termination mode)